wallet/wallet-rpc-lib/src/rpc/types.rs

```diff
@@ -376,3 +376,9 @@ pub struct RpcTokenId {
 pub struct NodeVersion {
     pub version: String,
 }
+
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+pub struct CreatedBlockInfo {
+    pub block_id: Id<GenBlock>,
+    pub block_height: BlockHeight,
+}
```

Comment:
Any particular reason you did not just use `BlockInfo`?

Reply:
fixed